CONTRIBUTING.md

```diff
@@ -12,119 +10,81 @@ We'd love for you to contribute to our source code and to make Accord Project do
 * [Updating Documentation][contribute.updating]
 * [Issue Submission Guidelines][contribute.submit]
 * [Pull Request Submission Guidelines][contribute.submitpr]
+* [Technical Documentation][contribute.techdocs]
 
 ## <a name="coc"></a> Code of Conduct
 
-Help us keep the Accord Project open and inclusive. Please read and follow our [Code of Conduct][coc].
+Help us keep the Accord Project open and inclusive. Please read and follow our [Code of Conduct][coc], as we expect project participants to adhere to it. Please read the full text so that you can understand what actions will and will not be tolerated.
```

Comment:
That's a little strong? Is there language from the [coc] we could use here?

Reply:
Yeah I can just remove this and revert to how it was.